Tests/TensorFlowTests/OperatorTests/BasicTests.swift

```diff
@@ -707,6 +719,7 @@ final class BasicOperatorTests: XCTestCase {
         ("testConcatenation", testConcatenation),
         ("testVJPConcatenation", testVJPConcatenation),
         ("testTranspose", testTranspose),
+        ("testReverse", testReverse),
```

Comment:
Minor: it would be nice for tests to be named:
* `testTranspose` -> `testTransposed`
* `testReverse` -> `testReversed`

Reply:
Also changed `testTile`, `testReshape`, `testFlatten`.
I'm not sure if I should rename some tests (`testReshapeToScalar`, `testBroadcast3x0To3x3`, etc) so I left them.

Reply:
The renaming if necessary should happen in another PR. It's unrelated to this one.

Reply:
@Shashi456 
I reverted them except `testReversed`.
I'm glad if someone make other changes and PR (I'm not confident in my naming).